### ISSUE TITLE
Benchmarking CI run is silently failing

### ISSUE THREAD

**Issue by kclowes** [ethereum/web3.py#2234]
### What was wrong?

The Benchmarking suite is has these errors on CI: 
```
asyncio.exceptions.TimeoutError
Task exception was never retrieved
```

But is still "passing". An example can be seen [here](https://app.circleci.com/pipelines/github/ethereum/web3.py/1920/workflows/0dae272b-fac5-4713-b1d0-01f1292d9f54/jobs/177486).

### How can it be fixed?

I haven't dug into when or why this is happening, but wanted to leave a placeholder.

**Comment by dbfreem** [comment]
Yea the issue is that on the benchmark for 100 request the geth node stops responding at around 80 request. That is when the timeouts come in because the node doesn't ever respond. 

I stopped the debugger after the first timeout. I made a manual sendTransaction requester to the test node while it was still running. It never responded, but other request types did respond. So this seems like an issue with the geth node or maybe the flags being used to start the test node.
